### ISSUE TITLE
Error: Unsupported Target

### ISSUE THREAD

**Issue by kylej-89** [Azure/azure-mobile-apps-js-client#294]
Not sure what I am missing for this, I am getting and Uncaught Error: Unsupported Target in index.js

var target = require('./environment').getTarget();

This returns "Unknown". This is installed in an Angular 8 application, i'm not sure how to get it to be of target type "Web"

**Comment by annakocheshkova** [comment]
Hi @kylej-89! Is it a mobile app or a website? 
Could you check the following: in console, if you type `global` you will most likely get `undefined` (that's why the error), while `window` or `globalThis` should return objects. Am I correct?

**Comment by kylej-89** [comment]
Hi, thanks for the quick reply, it is a website right now, but eventually will be a mobile app. You're right, global is not defined and window and globalThis are defined. 

Is there a workaround to this?

**Comment by annakocheshkova** [comment]
Yes, that's all I needed at this point, thank you. This should be fixed in the sdk, however if you are blocked right now, you can [edit this line](https://github.com/Azure/azure-mobile-apps-js-client/blob/46a25c82c7bbe9299c4c1b6bd70056a308ac7d0d/sdk/src/Platform/environment.js#L16) to be 
`} else if (typeof global !== 'undefined' || typeof globalThis !== 'undefined' || typeof window !== undefined) {`

**Comment by PaulARoy** [comment]
Will there be any update on npm for this fix? It's a bit tricky to use the SDK in the several environments and would be cleaner to add through npm.

**Comment by Jamminroot** [comment]
Hey, @PaulARoy, there are currently no updates planned

**Comment by russelarms** [comment]
@PaulARoy, unfortunately, we are not planning to release a new version of the SDK anytime soon.

**Comment by PaulARoy** [comment]
Thank you for the answer!
